pkg/sql/app_stats.go

```diff
@@ -349,6 +353,14 @@ func (e *Executor) ResetStatementStats(ctx context.Context) {
 	e.sqlStats.resetStats(ctx)
 }
 
+// LasStatementStatReset returns the time the stmt stats were last rest.
+func (e *Executor) LasStatementStatReset() time.Time {
+	e.sqlStats.Lock()
```

Comment:
I don't think this does what you think it does... 
The `sqlStats` you want I believe are these: https://github.com/cockroachdb/cockroach/blob/58335bcaa3faa1ff27981c22e27b998125470983/pkg/sql/conn_executor.go#L229

I see there is a test change in this commit; does that prove that I'm confused?

Generally, the `Executor` is mostly unused code, so it shouldn't be touched.

Reply:
Addressed in #24941

Reply:
We should really nuke Executor if it is dead code.

Reply:
trying, trust me